drivers/frequency/adf4377/adf4377.c

```diff
@@ -31,11 +32,73 @@
  * EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 *******************************************************************************/
 
+/******************************************************************************/
+/***************************** Include Files **********************************/
```

Comment:
we decided to drop these headers lately, see https://github.com/analogdevicesinc/no-OS/pull/2468